pkg/validator/validator.go

```diff
@@ -0,0 +1,13 @@
+package validator
+
+import (
+	"strings"
+
+	"github.com/crisBrunette/questioncicas/internal/database"
+	"github.com/crisBrunette/questioncicas/pkg/question_generator"
+)
+
+func Validator(userAnswer string) bool {
+	correctAnswer, _ := database.QuestionAnswer[question_generator.RamdonQuestion]
+	return strings.EqualFold(userAnswer, correctAnswer)
```

Comment:
honestly first time I see the usage of strings.EqualFold hahahah.
You can just do userAnswer == correctAnswer.
It's true that it's not exactly the same thing since in your case you are also making it case insensitive which tbh is better.
But nice nonetheless.